ext/standard/basic_functions_arginfo.h

```diff
@@ -2175,6 +2175,9 @@ ZEND_END_ARG_INFO()
 
 #define arginfo_memory_get_peak_usage arginfo_memory_get_usage
 
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_memory_reset_peak_usage, 0, 0, IS_VOID, 0)
```

Comment:
This change looks suspicious to me: the stub file from which this file is generated is not modified. I suspect that this was changed manually instead of being generated, which would then be lost on next run of the generator.

Reply:
And you were [so right](https://ci.appveyor.com/project/php/php-src/builds/43100993/job/gkjghgmictyu8pa8?fullLog=true#L4067)! I'm going to fix this.

Reply:
can `git add . -N && git diff --exit-code` be added to CI to check if all generated files are in sync after build is run?

Reply:
That would make sense. Would also prevent adding malicious code to generated files like zend_vm_execute.h.